Dockerfile

```diff
@@ -17,24 +17,22 @@ RUN apt-get update -y \
 
 WORKDIR /usr/src/ferdium
 
+RUN npm i -gf "pnpm@$(node -p 'require("./package.json").engines.pnpm')" && pnpm -v
```

Comment:
This line doesn't seems to work, because `package.json` is not copied into the container at this point.

Reply:
nice catch @nathanaelhoun - fixed it in a new commit